backend/src/main/java/ai/verta/modeldb/artifactStore/storageservice/nfs/NFSService.java

```diff
@@ -243,4 +243,9 @@ public String generatePresignedUrl(String artifactPath, String method) {
       throw StatusProto.toStatusRuntimeException(status);
     }
   }
+
+  @Override
+  public void initializeMultipart(String s3Key) {
```

Comment:
We will not be supporting it in NFS for the time being.

Reply:
changed that it will return null that will mean failed precondition